releases/cloud/point-2024-06-04-release-note.md

```diff
@@ -0,0 +1,44 @@
+---
+description: This article lists improvements and bug fixes in the Syskit Point Cloud version 2024.3.50.15
+---
+
+# June 04, 2024
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business.
+
+
+## Improvements & Bug Fixes
+
+* **Improvements made to Storage Reports!**
+  * **The Storage Metrics report** has a new column added - Version Limit - where you can see if a workspace has a Syskit Point or Microsoft 365 version limit assigned, as well as which version limit is applied to which workspace. 
+  * **The Site Storage Metrics** report has a new column added - Last Modified - where you can see when a file was last modified.
+
+* **The Private Workspaces Shared with Everyone** policy can now be resolved manually or resolved automatically.
```

Comment:
@ivicabosk91  ovo je iza feature flaga ili ide publish?

Reply:
Feature flag je ovo, nije to još gotovo, samo jedan dio je ušao u Trunk tako da ovaj dio treba obrisati iz Release notesa (linija 23)